CMakeLists.txt

```diff
@@ -70,52 +64,57 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# Set the minimum python version to 3.7
+set(PYBIND11_PYTHON_VERSION 3.7)
 
-unset(PYTHON_EXECUTABLE)
-find_package(Python 3.7 COMPONENTS Interpreter Development REQUIRED)
-set(PYTHON_INCLUDE_DIRS ${Python_INCLUDE_DIR})
-set(PYTHON_EXECUTABLE ${Python_EXECUTABLE})
-set(PYBIND11_PYTHON_VERSION ${Python_VERSION})
-find_package(pybind11 QUIET)
-if(NOT pybind11_FOUND)
-  message("-- Cannot build the python module 'Sofa', missing the pybind software to generate the bindings.")
-  return()
-endif()
+# Set the minimum pybind11 version to 2.3 (before that the pybind11::embed target did not exist)
+find_package(pybind11 2.3 CONFIG QUIET REQUIRED)
```

Comment:
The readme for SofaPython3 says we handle pybind11 > 2.2.4. Maybe we should either change the readme of change the version here.

Reply:
I will change the readme, like the comment says, this won't work with 2.2.4 since it is missing a cmake target that we use for the Plugin.

Reply:
OK =)
Given how old this PR is I suggest we merge as soon as the readme is updated